pkgs/applications/graphics/gimp/default.nix

```diff
@@ -135,84 +173,144 @@ stdenv.mkDerivation (finalAttrs: {
       ghostscript
       aalib
       shared-mime-info
+      json-glib
       libwebp
       libheif
+      python
       libexif
       xorg.libXpm
+      xorg.libXmu
       glib-networking
       libmypaint
       mypaint-brushes1
+
+      # New file dialogue crashes with “Icon 'image-missing' not present in theme Symbolic” without an icon theme.
+      adwaita-icon-theme
+
+      # for Lua plug-ins
+      (luajit.withPackages (pp: [
+        pp.lgi
+      ]))
+    ]
+    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [
+      alsa-lib
+
+      # for JavaScript plug-ins
+      gjs
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       AppKit
       Cocoa
-      gtk-mac-integration-gtk2
     ]
-    ++ lib.optionals stdenv.hostPlatform.isLinux [
+    ++ lib.optionals stdenv.isLinux [
       libgudev
-    ]
-    ++ lib.optionals withPython [
-      python
-      # Duplicated here because python.withPackages does not expose the dev output with pkg-config files
-      python2.pkgs.pygtk
     ];
 
-  # needed by gimp-2.0.pc
   propagatedBuildInputs = [
+    # needed by gimp-3.0.pc
     gegl
+    cairo
+    pango
+    gexiv2
   ];
 
-  configureFlags =
+  mesonFlags =
     [
-      "--without-webkit" # old version is required
-      "--disable-check-update"
-      "--with-bug-report-url=https://github.com/NixOS/nixpkgs/issues/new"
-      "--with-icc-directory=/run/current-system/sw/share/color/icc"
-      # fix libdir in pc files (${exec_prefix} needs to be passed verbatim)
-      "--libdir=\${exec_prefix}/lib"
+      "-Dbug-report-url=https://github.com/NixOS/nixpkgs/issues/new"
+      "-Dicc-directory=/run/current-system/sw/share/color/icc"
+      "-Dcheck-update=no"
+      # Not very important to do downstream, save a dependency.
+      "-Dappdata-test=disabled"
+      # Not yet packaged.
+      "-Dilbm=disabled"
```

Comment:
Is it this https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/li/libilbm/package.nix ?